build/tools/cosmovisor.md

```diff
@@ -0,0 +1,55 @@
+# Cosmovisor
+
+_This doc is to be reviewed and updated_
+
+**cosmovisor** is a small process manager for any Cosmos SDK based application binaries that monitors the governance module for incoming **chain upgrade proposals**. If it sees a proposal that gets approved, cosmovisor can automatically download the new binary, stop the current binary, switch from the old binary to the new one, and finally restart the node with the new binary.
+
+For detailed information related to cosmovisor, refer here: [cosmos-sdk doc](https://docs.cosmos.network/main/tooling/cosmovisor).
+
+## Auto-Download feature
+
+If the system administrator plans the Persistence chain upgrade using Cosmovisor, generally the upgrade and current binaries need to be manually placed on the disk before the upgrade happens.  
```

Comment:
By system administrator you mean the node operator, right? 

Would suggest to change to: 
"If the node operator has installed Cosmovisor to facilitate the chain upgrades, ..."